bin/swan-configure.js

```diff
@@ -15,11 +15,8 @@ const questions   = {
     message : 'What is your github username:',
     default : (config.git) ? config.git.user.name : '',
     validate: function (input) {
-      if (input && input.length >= 5) {
-        return true;
-      }
-
-      return `Expected a valid github username.`;
+      let valid = typeof input === 'string' && input.length > 0;
```

Comment:
Also something a linter might be able to catch (at least in some cases). I get your point, and it's unfortunate that `inquirer` expect different types in the first place.